crates/cxx-qt-lib/src/gui/qpen.rs

```diff
@@ -129,10 +129,7 @@ mod ffi {
 
 #[repr(C)]
 pub struct QPen {
-    #[cfg(cxxqt_qt_version_major = "5")]
     _cspec: MaybeUninit<[i32; 2]>,
```

Comment:
why is this the size of two i32s ? it looks to be a single raw pointer in C++ ? So this should be `MaybeUninit<usize>` ?